uptane-standard.md

```diff
@@ -72,196 +72,462 @@ This document describes a framework for securing automotive software update syst
 
 An Uptane implementation SHALL make the following services available to vehicles:
 
 * Image repository
 * Director repository
 * Time server
 
-### Image Repository
+### **5.3.1 Image Repository**
 
-The Image repository exists to allow an OEM and/or its suppliers to upload images and their associated metadata. It makes these images and their metadata available to vehicles. The Image repository is designed to be primarily controlled by human actors, and updated relatively infrequently.
+The Image repository exists to allow an OEM and/or its suppliers to upload images and
+their associated metadata. It makes these images and their metadata available to 
+vehicles. The Image repository is designed to be primarily controlled by human actors,
+and updated relatively infrequently.
 
-The Image repository SHALL expose an interface permitting the download of metadata and images. This interface SHOULD be public.
+The Image repository SHALL expose an interface permitting the download of metadata
+and images. This interface SHOULD be public.
 
 The Image repository SHALL require authorization for writing metadata and images.
 
-The Image repository SHALL provide a method for authorized users to upload images and their associated metadata. It SHALL check that a user writing metadata and images is authorized to do so for that specific image by checking the chain of delegations for the image as described in {{delegations_meta}}.
+The Image repository SHALL provide a method for authorized users to upload images
+and their associated metadata. It SHALL check that a user writing metadata and images
+is authorized to do so for that specific image by checking the chain of delegations
+for the image as described in {{delegations_meta}}.
 
-The Image repository SHALL implement storage which permits authorized users to write an image file using a unique filename, and later read the same file using the same name. It MAY use any filesystem, key-value store, or database that fulfills this requirement.
+The Image repository SHALL implement storage which permits authorized users to
+write an image file using a unique filename, and later read the same file using
+the same name. It MAY use any filesystem, key-value store, or database that
+fulfills this requirement.
 
 The Image repository MAY require authentication for read access.
 
-### Director Repository
+### **5.3.2. Director Repository**
 
-The Director repository instructs ECUs as to which images should be installed by producing signed metadata on demand. Unlike the Image repository, it is mostly controlled by automated, online processes. It also consults a private inventory database containing information on vehicles, ECUs, and software revisions.
+The Director repository instructs ECUs as to which images should be installed
+by producing signed metadata on demand. Unlike the Image repository, it is
+mostly controlled by automated, online processes. It also consults a private
```

Comment:
Where should we discuss that the Director repository primarily uses online keys?